commonknowledge/mapped
# Feature/map 220 ensure action network webhooks work

Implements get_member_ids_from_webhook for ActionNetworkSource, which makes
webhooks work. Action Network only has webhooks for "action taken" and "new subscriber",
so to test this you must create a new activist in the action network group.

## Motivation and Context
https://linear.app/commonknowledge/issue/MAP-220/ensure-action-network-webhooks-work

## How Can It Be Tested?
1. Add an Action Network Data Source
2. Add a data mapping (e.g. for constituency)
3. If testing locally, use ngrok to get a public URL for the mapped backend
4. Enable Webhooks
5. Copy the Webhook URL into the Action Network API & Sync settings
6. Enable the Webhook on AN's side
7. Add a new activist
8. Wait for the webhook to be received and the queue job to run
9. Verify the custom field is set
